src/BenchmarkDotNet/Columns/FirstCallColumn.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
+namespace BenchmarkDotNet.Columns
+{
+    public class FirstCallColumn : IColumn
+    {
+        public string Id => nameof(FirstCallColumn);
+
+        public string ColumnName => "First Call";
+
+        public bool AlwaysShow => true;
+
+        public ColumnCategory Category => ColumnCategory.Custom;
```

Comment:
I think it will be better to use the `Statistics` category with the "latest" priority. From my point of view, this column may be considered as one of the statistics columns, it shouldn't be placed after baseline columns.